stalker/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Does the code of conduct really need to be in this directory ?

Reply:
No, I will remove it. Came from the automatic generation of the Gem.